recipes/cvplot/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class CvPlotConan(ConanFile):
+    name = "cvplot"
+    description = "fast modular opencv plotting library"
+    license = "MIT"
+    topics = ("plot", "opencv", "diagram", "plotting")
+    homepage = "https://github.com/Profactor/cv-plot"
+    url = "https://github.com/conan-io/conan-center-index"
+    generators = "cmake"
```

Comment:
Seems unused

```suggestion
```

Reply:
You are right, we only need cmake in test_package.